packages/db/src/loaders/commands/compile.ts

```diff
@@ -0,0 +1,144 @@
+import { TruffleDB } from "@truffle/db/db";
+import {
+  WorkflowCompileResult,
```

Comment:
It looks like some of these types now exist in `compile-common`. See [here](https://github.com/trufflesuite/truffle/blob/develop/packages/compile-common/src/types.ts).

Reply:
Merged changes to address this!